go.mod

```diff
@@ -10,4 +10,12 @@ require (
 	github.com/libp2p/go-libp2p-core v0.3.0 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.4.0
+	github.com/tj/go-spin v1.1.0 // indirect
```

Comment:
@cryptonemo - Suggestion: It looks like this stuff was added erroneously. If that's true, run `go mod tidy` to clean this (and `go.sum`) up.